dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -552,15 +552,18 @@ public void save(final Folder folder, final String existingId,
 
 		Host host = APILocator.getHostAPI().find(folder.getHostId(), user, respectFrontEndPermissions);
 		Folder parentFolder = findFolderByPath(id.getParentPath(), id.getHostId(), user, respectFrontEndPermissions);
-		Permissionable parent = id.getParentPath().equals("/")?host:parentFolder;
+		Permissionable parent = "/".equals(id.getParentPath()) || "/System folder".equals(id.getParentPath()) 
+				        ? host : parentFolder;
 
 		if(parent ==null){
 			throw new DotStateException("No Folder Found for id: " + id.getParentPath());
 		}
 		if (!permissionAPI.doesUserHavePermission(parent, PermissionAPI.PERMISSION_CAN_ADD_CHILDREN, user,respectFrontEndPermissions)
 				|| !permissionAPI.doesUserHavePermissions(PermissionableType.FOLDERS, PermissionAPI.PERMISSION_EDIT, user)) {
-			String userId = user.getUserId() != null ? user.getUserId() : "";
-			throw new AddContentToFolderPermissionException(userId, parentFolder.getPath());
+					final String userId = user.getUserId() != null ? user.getUserId() : "";
+					final String parentFolderAsString = "/".equals(id.getParentPath()) || "/System folder".equals(id.getParentPath()) 
```

Comment:
Would be nice to create constants for **"/System folder"** and **"/"**

Reply:
I had them on the original PR, so let me update this one as well